PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -519,8 +519,10 @@ def produceMET(self, process,  metType, metModuleSequence, postfix):
             getattr(process, _myPatMet).srcPFCands = "puppiForMET" if self.getvalue("Puppi") else copy.copy(self.getvalue("pfCandCollection"))
         if metType == "PF":
             from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
-            run2_miniAOD_UL.toModify(getattr(process, _myPatMet), srcLeptons = \
-              cms.VInputTag(copy.copy(self.getvalue("electronCollection")) if self.getvalue("onMiniAOD") else
+            from Configuration.Eras.Modifier_run2_nanoAOD_106Xv1_cff import run2_nanoAOD_106Xv1
+            for modifier in run2_miniAOD_UL, run2_nanoAOD_106Xv1:
```

Comment:
The  run2_miniAOD_UL was introduced multiple times in the https://github.com/cms-sw/cmssw/pull/29392.
why it's applied now only for the "PF" case ?